features/step_definitions/trip.js

```diff
@@ -35,11 +34,12 @@ module.exports = function () {
                     }
 
                     if (headers.has('status')) {
-                        got.status = json.status.toString();
+                        got.status = json.code;
```

Comment:
The status.toString() code was copy pasted from another cucumber test but wasn't really being used in the trip context (it was broken code essentially with no way to trigger it to check), so I changed it so that it does work.